docs/src/APIs/canopy/Photosynthesis.md

```diff
@@ -47,12 +47,10 @@ ClimaLand.Canopy.make_PModel_callback
 ClimaLand.Canopy.intrinsic_quantum_yield
 ClimaLand.Canopy.compute_viscosity_ratio
 ClimaLand.Canopy.compute_Kmm
-ClimaLand.Canopy.optimal_co2_ratio_c3
 ClimaLand.Canopy.intercellular_co2_pmodel
 ClimaLand.Canopy.gs_co2_pmodel
 ClimaLand.Canopy.gs_h2o_pmodel
 ClimaLand.Canopy.compute_mj
-ClimaLand.Canopy.compute_mc
```

Comment:
We do still have this function (in src/standalone/Vegetation/pmodel.jl). There's an extra line between the docstring and the function so maybe it didn't show up